firstvoices/backend/tests/test_apis/test_join_request_api.py

```diff
@@ -0,0 +1,61 @@
+from backend.models import JoinRequest
+from backend.models.join_request import JoinRequestReason, JoinRequestStatus
+from backend.tests import factories
+from backend.tests.test_apis.base_api_test import (
+    BaseReadOnlyUncontrolledSiteContentApiTest,
+)
+
+
+class TestJoinRequestEndpoints(BaseReadOnlyUncontrolledSiteContentApiTest):
+    """
+    End-to-end tests that the join request endpoints have the expected behaviour.
+    """
+
+    API_LIST_VIEW = "api:join-request-list"
+    API_DETAIL_VIEW = "api:join-request-detail"
+
+    model = JoinRequest
+
+    def get_expected_standard_fields(self, instance, site):
```

Comment:
Given that these are standard, should they be defined in the base class rather than in each api test?

Reply:
This model is an exception to the standard as it does not have the `title` property. What has been defined here is every field outside of title

Reply:
I've opted to not used that function in this test set as it does not apply to this particular model